src/Microsoft.OData.Edm/Validation/EdmErrorLevel.cs

```diff
@@ -0,0 +1,28 @@
+﻿//---------------------------------------------------------------------

```

Comment:
Any reason why you settled on these 3?
You could take a look at the [LogLevel Enum](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loglevel?view=dotnet-plat-ext-3.1) as used in the Microsoft.Extensions.Logging lib...

Reply:
+1, but need to figure out a subset. 

---
In reply to: [420867935](https://github.com/OData/odata.net/pull/1758#discussion_r420867935) [](ancestors = 420867935)

Reply:
I settled for  
Error = 3
Information = 1
None = 0
Warning = 2 , None being default.